docs/docs/07-Development/01-setup.md

```diff
@@ -60,4 +60,13 @@ In dev mode, opening and closing the plugin menu should reload the code.
 
 ## Docker Dev Env
 
-If the manual setup is too much hassle for you. You can use a docker based dev environment by running `docker compose -f docker/docker-compose.dev.yml up` in the root of the repo. This setup wasn't super reliable for me though.
+If the manual setup is too much hassle for you, you can use a docker based dev environment instead by following the instructions:
+
+- Clone the repo and navigate to the root of the repo.
+- Run `cp /docker/.env.sample /docker/.env`
```

Comment:
I don't think there's a need for an env file during development to be honest. Hoarder should work just fine without one.

Reply:
This ties back into the above comment: "Wouldn't it be a good idea to set it up anyways, just to make sure that any development does not break functionality offered by nextauth or meilisearch?"

Reply:
coming back to this: wouldnt we need it for NEXTAUTH_URL and MEILISEARCH_URL?